### PR TITLE
Update index.tsx to add matrix to footer

### PR DESCRIPTION
**Pull request by SandroLinux**
Update index.tsx to add matrix to footer

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -32,6 +32,9 @@ export const Footer = () => {
                 <a href={'https://dothq.link/dsc'} target={'_blank'}>
                   <i className={'discord-icon'}></i>
                 </a>
+                <a href={'https://dothq.link/matrix'} target={'_blank'}>
+                <i className={'matrix-icon'}></i>
```
Comment (EnderDev) [inline src/components/Footer/index.tsx:36]: Can we indent this?

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -32,6 +32,9 @@ export const Footer = () => {
...
+              </a>
```
Comment (EnderDev) [inline src/components/Footer/index.tsx:37]: This too please.

**Review by EnderDev — CHANGES_REQUESTED** [review]

File: src/components/Footer/index.tsx (hunk 2 above)
Comment (SandroLinux) [reply src/components/Footer/index.tsx:37]: ok

Comment (SandroLinux) [reply src/components/Footer/index.tsx:37]: I will open up a new pull request with the requested changes

Comment (EnderDev) [reply src/components/Footer/index.tsx:37]: Why? Just make your changes to this one.

**Review by SandroLinux — COMMENTED** [review]
Ok yeah

**Comment by trevorthalacker** [comment]
closing as #17 fixed this

**Pull request closed without merging**